.github/workflows/ruby.yml

```diff
@@ -0,0 +1,46 @@
+name: Ruby
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version:
+        - jruby
+        - 2.4.10
+        - 2.5.8
+        - 2.6.6
+        - 2.7.1
+        - 2.7.2
+        gemfile:
+        - gemfiles/rails5_0.gemfile
+        - gemfiles/rails5_1.gemfile
+        - gemfiles/rails5_2.gemfile
+        - gemfiles/rails6_0.gemfile
+        exclude:
+          - ruby-version: 2.4.10
+            gemfile: gemfiles/rails6_0.gemfile
+    env:
+      JRUBY_OPTS: "--1.9"
+      BUNDLE_GEMFILE: "${{ matrix.gemfile }}"
+    steps:
+    - name: Git checkout
+      uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: "${{ matrix.ruby-version }}"
+    - name: Start MongoDB
+      uses: supercharge/mongodb-github-action@1.3.0
+      with:
+        mongodb-version: 4.4
```

Comment:
This time I did not prepare for multiple mongodb versions to test, but we can easily add another matrix for mongodb versions too.